src/commands/LyricsCommand.ts

```diff
@@ -0,0 +1,59 @@
+import { BaseCommand } from "../structures/BaseCommand";
+import { IMessage } from "../../typings";
+import { DefineCommand } from "../utils/decorators/DefineCommand";
+import { createEmbed } from "../utils/createEmbed";
+import { isUserInTheVoiceChannel, isSameVoiceChannel, isValidVoiceChannel } from "../utils/decorators/MusicHelper";
+import fetch from "node-fetch";
+
+@DefineCommand({
+    aliases: ["ly"],
+    name: "lyrics",
+    description: "Get lyrics from the current song",
+    usage: "{prefix}lyrics"
+})
+export class LyricsCommand extends BaseCommand {
+    @isUserInTheVoiceChannel()
+    @isValidVoiceChannel()
+    @isSameVoiceChannel()
+    public async execute(message: IMessage): Promise<void> {
+        const song = message.guild?.queue?.songs.first()?.title;
+        if (!message.guild?.queue?.songs) {
+            message.channel.send(createEmbed("error", "There is nothing playing")).catch(e => this.client.logger.error("LYRICS_CMD_ERR:", e));
+        } else if (song) {
+            const url = `https://api.lxndr.dev/lyrics?song=${encodeURI(song)}`;
+            fetch(url)
+                .then(response => response.json())
+                .then(data => {
+                    let lyrics = data.lyrics;
+                    let album_art = data.album_art;
+                    const charLength = lyrics.length;
+                    let cantEmbeds = 0;
+
+                    if (charLength < 2048) {
+                        cantEmbeds = 1;
+                    } else {
+                        for (let i = 2; i < 10; i++) {
+                            if (charLength < 2048 * i) {
+                                cantEmbeds = i;
+                                break;
+                            }
+                        }
+                    }
+                    if (!album_art) album_art = "https://kknd26.ru/images/no_photo.png";
```

Comment:
Use Zhycorp's logo instead for no photo
```suggestion
                    if (!album_art) album_art = "https://api.zhycorp.com/assets/images/logo.png";
```